openlibrary/admin/stats.py

```diff
@@ -120,10 +120,12 @@ def main(infobase_config, openlibrary_config, coverstore_config, ndays = 1):
 
     setup_ol_config(openlibrary_config)
 
-    # Gather delta and total counts
-    # Total counts are simply computed and updated for the current day
-    # Delta counts are computed by subtracting the current total from yesterday's total
-    today = datetime.datetime.now()
+    """
+      Gather delta and total counts
+      Total counts are simply computed and updated for the current day
+      Delta counts are computed by subtracting the current total from yesterday's total
+    """
+   today = datetime.datetime.now()
```

Comment:
This will be an IndentationError which has the potential to halt the program on our users.